### PR TITLE
Adjusts medibot heal thresholds.

### PR DESCRIPTION
**Pull request by AcapitalA**
## About The Pull Request

Medibot healing thresholds have been adjusted:

From 95 / 25 HP.
To 100 / -95 HP.

## Why It's Good For The Game

Allows medibots to actually top you off without RnD upgrades and luck.

## Changelog
:cl:
qol: Medibots now have wider healing thresholds you can set it at.
/:cl:

### CODE REVIEW

**Comment by Sparkezel** [comment]
Everyone that hurt their pinky is going to be chased by medibot this way, are you sure you want it to start from 0?

**Comment by AcapitalA** [comment]
Yes, because it doesn't START at zero, now you just get the ability to SET it to 0 if you modify it in-game.

They still spawn at ten.

**Comment by Sparkezel** [comment]
good then

File: code/modules/mob/living/simple_animal/bot/medbot.dm [hunk 1]
```diff
@@ -180,10 +180,10 @@
...
 		var/adjust_num = text2num(href_list["adj_threshold"])
 		heal_threshold += adjust_num
-		if(heal_threshold < 5)
-			heal_threshold = 5
-		if(heal_threshold > 75)
-			heal_threshold = 75
+		if(heal_threshold < 0)
+			heal_threshold = 0
+		if(heal_threshold > 195)
+			heal_threshold = 195
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/simple_animal/bot/medbot.dm:186]:
```suggestion
		heal_threshold = clamp(heal_threshold, 0, 195)
```

**Comment by AcapitalA** [comment]
Mergies please.

**Pull request closed without merging**